app/graphql/util/graphql_policy.rb

```diff
@@ -85,6 +85,14 @@ class GraphqlPolicy
 
           current_user.admin? || current_user.admin_of?(team)
         end,
+        setActiveKudosMeter: ->(_obj, args, ctx) do
+          current_user = ctx[:current_user]
+          return false unless current_user.present?
+
+          team = Team.find(args[:teamId])
+
+          current_user.admin? || current_user.admin_of?(team)
```

Comment:
There are two admin roles within the system which makes this a bit confusing. There are systemwide admins, which is a boolean stored within the user model and you have team admins. These are defined by the role string in the TeamMember class. I think refactoring this would require some work and is something for another PR.